internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -145,42 +334,64 @@ func (s *Server) HandleGitHubAppWebhook() http.HandlerFunc {
 		}
 
 		wes.Typ = github.WebHookType(r)
-		if wes.Typ == "ping" {
-			logPingReceivedEvent(r.Context(), rawWBPayload)
-			wes.Error = false
-			return
-		}
 
 		m := message.NewMessage(uuid.New().String(), nil)
 		m.Metadata.Set(events.ProviderDeliveryIdKey, github.DeliveryID(r))
-		m.Metadata.Set(events.ProviderSourceKey, "https://api.github.com/") // TODO: handle other sources
+		// TODO: handle other sources
+		m.Metadata.Set(events.ProviderSourceKey, "https://api.github.com/")
 		m.Metadata.Set(events.GithubWebhookEventTypeKey, wes.Typ)
 
-		ctx := r.Context()
 		l := zerolog.Ctx(ctx).With().
 			Str("webhook-event-type", m.Metadata[events.GithubWebhookEventTypeKey]).
 			Str("providertype", m.Metadata[events.ProviderTypeKey]).
 			Str("upstream-delivery-id", m.Metadata[events.ProviderDeliveryIdKey]).
 			Logger()
+		ctx = l.WithContext(ctx)
+
+		wes.Accepted = true
```

Comment:
Question: Would it be safer to rely on how we already initialize `wes` here with `Accepted: false` and only set it to true when we know we accept the message instead of the other way around?

Reply:
I'll set `wes.Accepted` to true inside the switch case, looks clearer to me.